pico_project.py

```diff
@@ -1132,10 +1179,12 @@ def DoEverything(parent, params):
     # First check if there is already a project in the folder
     # If there is we abort unless the overwrite flag it set
     if os.path.exists(CMAKELIST_FILENAME):
-        if not params.wantOverwrite :
+        if not params.wantOverwrite:
             if params.wantGUI:
                 # We can ask the user if they want to overwrite
-                y = mb.askquestion('Raspberry Pi Pico Project Generator', 'There already appears to be a project in this folder. \nPress Yes to overwrite project files, or Cancel to chose another folder')
+                y = mb.askquestion(
+                    'Raspberry Pi Pico Project Generator',
+                    'There already appears to be a project in this folder. \nPress Yes to overwrite project files, or Cancel to chose another folder')
```

Comment:
Does this also go over your maximum line length? :shrug:

Reply:
I've set my max line length at 120 chars. I think it does go over in this case